src/js/client/renderer.js

```diff
@@ -97,7 +98,13 @@
             }
         },
         events: {
-            "onTemplatesLoaded": null
+            "onTemplatesLoaded": null,
+            "onAllResourcesLoaded": {
+                events: {
+                    // onMessagesLoaded: "onMessagesLoaded",
```

Comment:
This is a bit baffling, you still have the rollup event, and you still launch the template retrieval, but you've commented out the templatesLoaded event twice?